src/topology/connected.lean

```diff
@@ -1547,3 +1547,37 @@ lemma continuous.connected_components_map_continuous {β : Type*} [topological_s
 continuous.connected_components_lift_continuous (continuous_quotient_mk.comp h)
 
 end connected_component_setoid
+
+/-- A set `s` is preconnected if and only if every
+map into `bool` that is continuous on `s` is constant on `s` -/
```

Comment:
The docstring does not correspond to the content of the lemma. Docstrings appear as documentation attached to one single lemma, so you should give a docstring for this lemma adapated to its content, and another docstring for the next lemma.

Reply:
Fixed. Thanks!